src/manage_nsfs/manage_nsfs_cli_responses.js

```diff
@@ -31,7 +31,11 @@ class ManageCLIResponse {
                 code: this.code,
             }
         };
-        if (this.list || this.status) json.response.reply = typeof detail === 'string' ? JSON.parse(detail) : detail;
+        if (this.list || this.status) {
+            json.response.reply = typeof detail === 'string' ? JSON.parse(detail) : detail;
+        } else {
+            json.response.message = typeof detail === 'string' ? detail : `${detail}`;
+        }
```

Comment:
I think you need to explain this change in a comment.
From what I understand you're going to add a `ManageCLIResponse.` to all the responses which are not `list` or `status`, when the change is currently only related to `ManageCLIResponse.AccountDeleted` and `ManageCLIResponse.BucketDeleted`.

In my opinion, and maybe others would think otherwise I would add a property of `delete` (like list and status) and make the change only under this condition. WDYT?

Reply:
We can add message property here.